### ISSUE TITLE
Custom theming support

### ISSUE THREAD

**Issue by callil** [cabal-club/cabal-desktop#67]
see: https://github.com/hundredrabbits/Themes

**Comment by cblgh** [comment]
cc @simbolo for css structure thoughts :3

**Comment by nikolaiwarner** [comment]
In the long run, I'd like to see theming go as wild as enabling users to swap out entire components that can be shareable and remixable.

**Comment by callil** [comment]
yes! I was thinking about this today. May be good to start planning this
soon. I can imagine a kind of chat config page when setting up a cabal
where you and add threads, themes, bots, your own imported components etc
etc and really make your own structure for a chat app.
